sys/parsoid.js

```diff
@@ -159,11 +159,27 @@ function _dependenciesUpdate(hyper, req) {
     });
 }
 
+function compileReRenderBlacklist(blacklist) {
+    const result = {};
+    blacklist = blacklist || {};
+    Object.keys(blacklist).forEach((domain) => {
+        result[domain] = blacklist[domain].map((entry) => {
```

Comment:
Nice idea. Will modify. I think we have the code that does exactly that somewhere but I've completely forgotten where that code is...